### DISCUSSION TITLE
What/Where is this function makeface

### DISCUSSION THREAD

**Discussion by AnthonyB08** [idaholab/moose#23548] [Q&A Modules: Navier-Stokes] [answered]
Hi all,

I am posting to ask where this function type is located in the moose directories : 
```
INSFVMassAdvection::computeQpResidual()
{
  const auto v = _rc_vel_provider.getVelocity(_velocity_interp_method, *_face_info, _tid);
  const auto rho_face = _rho(makeFace(
      *_face_info, limiterType(_advected_interp_method), MetaPhysicL::raw_value(v) * _normal > 0));
  return _normal * v * rho_face;
}
```
I am assuming it is the density*surface area of face , but I'd like to see the hard-code for further understanding.

**Comment by GiudGiud** [comment, marked as answer]
See there
https://github.com/idaholab/moose/blob/next/modules/navier_stokes/src/fvkernels/INSFVMassAdvection.C

The surface_area multiplication is there:
https://github.com/idaholab/moose/blob/43ab083397558b061f3d32ddf74a16e41bcc9902/framework/src/fvkernels/FVFluxKernel.C#L129

Guillaume

**Reply by AnthonyB08** [reply]
I'm sorry, I should have been more specific.  Where is the "makeface(*_face_info, limiterType(_advected_interp_method), MetaPhysicL::raw_value(v) * _normal > 0))" function located?

**Reply by AnthonyB08** [reply]
I see it disregard - thanks
